packages/react/src/useUploadThing.ts

```diff
@@ -114,6 +116,12 @@ export const INTERNAL_uploadthingHookGen = <
         await opts?.onClientUploadComplete?.(res);
         return res;
       } catch (e) {
+        /**
+         * This is the only way to introduce this as a non-breaking change
+         * Consider refactoring API in the next major version
```

Comment:
```suggestion
         * TODO: Consider refactoring API in the next major version
```

Just makes it easier to find these